doc/CMakeLists.txt

```diff
@@ -16,6 +16,9 @@ endif()
 
 find_package(Doxygen)
 if(DOXYGEN_FOUND)
+  if (DOXYGEN_VERSION VERSION_LESS 1.10.0)
+    message(WARNING "Found doxygen ${DOXYGEN_VERSION}, expecting >= 1.10.0, documentation might be incorrect")
```

Comment:
Fixed.

Reply:
Major issue users can met with older Doxygen versions is broken MathJax support - here I switched to MathJax 3 which is only supported since Doxygen 1.9.2 (https://www.doxygen.nl/manual/changelog.html). Technically we can support older versions with MathJax 2 using cmake, but question is whether it is really necessary, or who might require such support. My argument against it is that Doxygen is a portable application and can be installed on any major system from an archive or, alternatively, users can patch Doxyfile.in for their needs.